### DISCUSSION TITLE
About `@Init(label: String)`

### DISCUSSION THREAD

**Discussion by WinterArch** [gohanlon/swift-memberwise-init-macro#37] [General]
`label:` May be Ambiguation?
[Specifying Argument Labels](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions#Specifying-Argument-Labels)

**Comment by gohanlon** [comment]
Thank you for bringing this up, @WinterArch. I'm not quite sure I understand your concern about ambiguity. Could you please provide an example or explain your thoughts in more detail?
